rules/ml/ml_linux_anomalous_metadata_user.toml

```diff
@@ -0,0 +1,30 @@
+[metadata]
+creation_date = "2020/09/22"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/22"
+
+[rule]
+anomaly_threshold = 75
+author = ["Elastic"]
+description = """
+Looks for anomalous access to the metadata service by an unusual user. The metadata service may be targeted in order to
```

Comment:
what is the metadata service?

Reply:
Ah I see from the issue. Should we be more specific in the description and name that it is AWS Metadata Service

Reply:
It actually is a feature of AWS, GCP and Azure. They all have it.

Reply:
maybe specify "cloud platform metadata service"?